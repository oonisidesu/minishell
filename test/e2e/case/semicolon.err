ls;
ls ;
ls | cat | cat;
ls | cat | cat ;
