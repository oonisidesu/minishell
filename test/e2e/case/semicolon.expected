minishell: ls;: command not found
ls: ;: No such file or directory
minishell: cat;: command not found
cat: ;: No such file or directory
